ElliotForWater/efw-webapp
# feature: Add Open search tags

## Description
The OpenSearch description format lets a website describe a search engine for itself, so that a browser or other client application can use that search engine. 
Adding those tags, after a user visit our website, we should get listed in the list of possible search engine to choose from, in major browsers and device.


#### Steps to Test or Reproduce
Visit the website and then go to browser default search engine list and check if EfW is listed among others.

------------------------------------------------
## PR Checklist:
